aws_s3_bucket.tf

```diff
@@ -61,31 +73,47 @@ resource "aws_s3_bucket" "s3VersioningMfaFalse" {
```

Comment:
**Accurics remediation**
 
**[HIGH]: S3 bucket should not host static website**
 
**Vulnerability**
Ensure that there are not any static websites being hosted on buckets you aren't aware of
 
**Remediation**
Follow this document to disable unwanted static websites: http://docs.aws.amazon.com/AmazonS3/latest/user-guide/static-website-hosting.html

source: accurics-